packages/ember-data/lib/adapters/rest-adapter.js

```diff
@@ -391,7 +391,7 @@ export default Adapter.extend(BuildURLMixin, {
     @return {Promise} promise
   */
   findQuery: function(store, type, query) {
-    var url = this.buildURL(type.typeKey, null, null, 'findQuery');
+    var url = this.buildURL(type.typeKey, query, null, 'findQuery');
```

Comment:
Thanks for passing `query` along to `buildURL`; I will find that useful!

I'm wondering, though, why is it passed as the `id` parameter and would it make more sense as the `snapshot` parameter? I know it's not really a snapshot, but it seems more like a snapshot than an id. Before this change, `id` is either `null`, a string, or an array of strings. With this change it can also be an object. `snapshot` on the other hand, is either `null`, an object, or an array of objects. Also, I think the `query` is used similarly as the snapshot (to fill arbitrary values in the url).

For reference, here is the signature of `buildURL`:

```javascript
buildURL: function(type, id, snapshot, requestType) {
```